dluhc-component-library/src/components/maps/DatasetLayers.tsx

```diff
@@ -0,0 +1,30 @@
+import { fromExtent } from "ol/geom/Polygon";
+import { useMemo } from "react";
+import { useFetchDatasets } from "src/api/planningData/api";
+import { useMap } from "src/contexts/mapContext";
+import DatasetLayer from "./DatasetLayer";
+
+interface DatasetLayersProps {
+  selectedDatasets: string[];
+}
+
+const DatasetLayers = ({ selectedDatasets }: DatasetLayersProps) => {
+  const map = useMap();
+  const { data } = useFetchDatasets();
+
+  const datasets = useMemo(
+    () =>
+      data?.filter((dataset) => selectedDatasets.includes(dataset.dataset)) ||
```

Comment:
When we are using an OR for returning one value or another rather than in the boolean manner, we should use `??`